tests/test_backend.py

```diff
@@ -3280,13 +3280,13 @@ def func(x, x_new_size_):
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: x_new_size})
 
     def test_adjust_contrast(self):
-        x_shape = [4, 3, 2]
-        x_val = np.arange(1, 1 + np.prod(x_shape), dtype=np.float32).reshape(x_shape)
-        y_val = np.array(2.1, np.float32)
         def func(x, y):
             x_ = tf.image.adjust_contrast(x, y)
             return tf.identity(x_, name=_TFOUTPUT)
-        self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: y_val})
+        for x_shape in [[4, 3, 2], [2, 3, 4, 5], [3, 4, 2, 4, 3]]:
```

Comment:
Is [3, 4, 2, 4, 3] a valid input for adjust_contrast() function? Ideally, the rank of image should be 4, right?

Reply:
It's a valid input. This function only requests input at least 3d, but doesn't require input to be 3d or 4d only.
https://www.tensorflow.org/api_docs/python/tf/image/adjust_contrast